src/main/java/com/google/firebase/auth/UserRecord.java

```diff
@@ -528,6 +532,17 @@ UpdateRequest setValidSince(long epochSeconds) {
       return this;
     }
 
+    UpdateRequest linkProvider(@NonNull UserProvider userProvider) {
```

Comment:
For our builders (which UpdateRequest effectively is), use `setProperty()` rather than `property()`. (So `setLinkProvider()` here.) Both for consistency with the existing type, (eg setValidSince) and also for consistency with our 'best practices' guide. (http://go/java-practices/builders#api, last bullet point.) Note that this directly conflicts with EJ (as pointed out by the practices doc.)